### PR TITLE
(WIP) tweaks to rails models and migrations lesson

### PR DESCRIPTION
**Pull request by jshawl**

### CODE REVIEW

**Comment by jshawl** [comment]
cc @andrewsunglaekim

**Comment by andrewsunglaekim** [comment]
Added some notes for clarity in parts of the plan. Feel free to update

Additionally, this pull request is taking out the curl and manually creating seeds file based on the you do exercises in the rails console. I think this is more valuable then just curling a massive seeds file. In the next class we can just give them seeds file in a repo. I'm not married to either if you prefer curling data in.

@jshawl

File: 05-mvc-with-rails/rails-models-and-migrations/readme.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ Use `rails c` to view the output of
 ENV["RAILS_ENV"]
 ```
 
+> You can see that the output is `developement`. ENV["RAILS_ENV"] is a way we get/set our environment and allows for different types of configurations. The other two types of environment are `test` and `production`
```
Comment (jshawl) [inline 05-mvc-with-rails/rails-models-and-migrations/readme.md]: development

File: 05-mvc-with-rails/rails-models-and-migrations/readme.md [hunk 2]
```diff
@@ -120,11 +138,12 @@ You can create the artists table and run this migration with `rake db:migrate`
...
 ## You do: view db/schema.rb
 
-The above command created this file. Take a minute to read through it. 
+The above command created this file. Take a minute to read through it.
 
 ## You do: Use Rails console
 
-to create at least two artists.
+- to create at least two artists.
+- save the commands you run in your text editor so you can leverage it later.
```
Comment (jshawl) [inline 05-mvc-with-rails/rails-models-and-migrations/readme.md]: where to save this file?

**Comment by andrewsunglaekim** [comment]
:+1:

**Comment by jshawl** [comment]
:+1:

**Pull request merged**